### ISSUE TITLE
Breakpoint in debugger does not work

### ISSUE THREAD

**Issue by rigerous** [eclipsesource/tabris-js#1362]
Breakpoint in debugger Google Dev Tools does not work.
I use tutorial: http://eclipsesource.com/blogs/2016/06/06/debugging-javascript-with-tabris-js/

- Device: Moto X Force, Android 6.0.1
- Tabris.js version: 2.0.2

![2017-05-28_23-21-03](https://cloud.githubusercontent.com/assets/14946265/26531908/70b9d09a-43fc-11e7-8e64-27e70fd3926f.png)

**Comment by irbull** [comment]
Thanks for the bug report. With the latest Tabris-Android client I can reproduce this too. It looks like it's actually a regression in our JavaScript runtime (J2V8). Since I'm the maintainer of that, I'll dig in there.

**Comment by irbull** [comment]
I have looked into this problem, and it appears to be related to the new version of J2V8 / V8. When trying to get the scriptName from a StackFrame we end up getting `null` instead. The debug tools need this value. I'm not sure if this is a bug in the new V8 or something that has changed and we need to adapt. I've opened https://github.com/eclipsesource/J2V8/issues/262 in J2V8 to track the problem there.
